### ISSUE TITLE
Return error for some player

### ISSUE THREAD

**Issue by qentin1741** [Hypixel-API-Reborn/hypixel-api-reborn#483]
### Bug report

Get error reading 'catacombs' for some uuid or player nickname but not for all

- Part of code where error occurred
`hypixel
	.getSkyblockProfiles('629bcf79-84cd-4866-835c-81838eb40398')
	.then((profiles) => {
		console.log(profiles);
	})
	.catch((e) => {
		console.error(e);
		// if player doesn't exist will throw an error
	});
`
- What error occurred
`TypeError: Cannot read properties of undefined (reading 'catacombs')
    at getDungeons (E:\dev\node_modules\hypixel-api-reborn\src\structures\SkyBlock\SkyblockMember.js:351:59)
    at new SkyblockMember (E:\dev\node_modules\hypixel-api-reborn\src\structures\SkyBlock\SkyblockMember.js:147:21)
    at E:\dev\node_modules\hypixel-api-reborn\src\structures\SkyBlock\SkyblockProfile.js:25:68
    at Array.map (<anonymous>)
    at new SkyblockProfile (E:\de\node_modules\hypixel-api-reborn\src\structures\SkyBlock\SkyblockProfile.js:25:57)
    at E:\dev\node_modules\hypixel-api-reborn\src\API\skyblock\getSkyblockProfiles.js:43:30
    at Array.map (<anonymous>)
    at module.exports (E:\de\node_modules\hypixel-api-reborn\src\API\skyblock\getSkyblockProfiles.js:43:19)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)`

**Comment by Ghosty920** [comment]
```js 
if (!objectPath.has(data, 'dungeons')) {
    return null;
}
```
Your profile data probably has like mine :
```json
"data": {
    "dungeons": {
        "dungeons_blah_blah": [
            "ophelia_first_talk"
        ]
    }
}
```
And when it calls `data.dungeons.dungeon_types.catacombs`, there's an exception because `dungeon_types` doesn't exist

**Comment by Kathund** [comment]
Fixed in #484
